geomesa-hbase/geomesa-hbase-datastore/src/test/scala/org/locationtech/geomesa/hbase/data/HBaseColumnGroupsTest.scala

```diff
@@ -125,10 +125,7 @@ class HBaseColumnGroupsTest extends HBaseTest with LazyLogging  {
     ds = DataStoreFinder.getDataStore(params.asJava).asInstanceOf[HBaseDataStore]
     ds.createSchema(sft)
     val writer = ds.getFeatureWriterAppend(sft.getTypeName, Transaction.AUTO_COMMIT)
-    features.foreach { f =>
-      FeatureUtils.copyToWriter(writer, f, useProvidedFid = true)
-      writer.write()
-    }
+    features.foreach(FeatureUtils.write(writer, _, useProvidedFid = true))
```

Comment:
Hmm... I wonder if we have a page of documentation to show 'best practices' like this.

Reply:
we don't, but I think if we wanted something like that we should maybe use a wiki page or just try to check during code reviews; I don't want to make them part of the 'public API' to where we have to worry about semver. That said, I'm sure a lot of downstream projects using geomesa could be a bit simpler using things like this.

Reply:
+1 to wiki or something like that.  Not a high priority, but yeah, good idea!